_ml-commons-plugin/api.md

```diff
@@ -115,6 +115,120 @@ The API returns information on the model, the algorithm used, and the content fo
 }
 ```
 
+## Upload load a model
+
+Use the upload operation to upload a pre-existing or custom model and into a model index. ML commons splits the model into smaller chunks and saves those chunks into the model's index.
+
+```json
+POST /_plugins/_ml/models/_upload
+```
+
+### Request fields
+
+Field | Data Type | Description
+:---  | :--- | :--- 
+`name`| string | The name of the model. |
+`version` | integer | The version number of the model. Version number should follow standard semantic format, major.minor.patch |
```

Comment:
Does the model version have to have the `major.minor.patch` format?

Reply:
I prefer that we only maintain a major version, which is typical of resource versioning control. What is the use case for maintaining a minor and patch version? 

Refer to MLFlow's versioning: https://www.mlflow.org/docs/latest/model-registry.html. They only maintain a major version.